examples/gnrc_border_router/Makefile

```diff
@@ -11,8 +11,8 @@ RIOTBASE ?= $(CURDIR)/../..
 UPLINK ?= ethos
 
 # Check if the selected Uplink is valid
-ifeq (,$(filter ethos slip wifi,$(UPLINK)))
-  $(error Supported uplinks are `ethos`, `slip` and `wifi`)
+ifeq (,$(filter ethos slip usb-cdc-ecm wifi,$(UPLINK)))
+  $(error Supported uplinks are `ethos`, `slip`, `usb-cdc-ecm` and `wifi`)
```

Comment:
Maybe just `cdc-ecm`? That's easier to type ;)

Reply:
12915f4ca